command/hook_ui.go

```diff
@@ -113,22 +126,59 @@ func (h *UiHook) PreApply(
 		operation,
 		attrString)))
 
+	// Set a timer to show an operation is still happening
+	time.AfterFunc(periodicUiTimer, func() { h.stillApplying(id) })
+
 	return terraform.HookActionContinue, nil
 }
 
+func (h *UiHook) stillApplying(id string) {
+	// Grab the operation
+	h.l.Lock()
+	state, ok := h.resources[id]
+	h.l.Unlock()
```

Comment:
Wondering if there is a tiny race condition here that would cause `Still ...` to print after the `complete` message.

Should this function hold the lock until after it prints?